core/controllers/base.py

```diff
@@ -148,7 +148,7 @@ class BaseHandler(webapp2.RequestHandler):
 
     URL_PATH_ARGS_SCHEMAS = None
     HANDLER_ARGS_SCHEMAS = None
-    ARGS_WHICH_DO_NOT_NEED_SCHEMA_VALIDATION = ['csrf_token']
+    ARGS_WHICH_DO_NOT_NEED_SCHEMA_VALIDATION = ['csrf_token', 'source']
```

Comment:
This change is same as change in PR: https://github.com/oppia/oppia/pull/13225 ,  The https://github.com/oppia/oppia/pull/13225#discussion_r663299957 explains the reason. As this was blocking sending post request to the controllers in the PR, added it here.
// cc @Nik-09